prebid-server/pbs-modules/rules-engine.md

```diff
@@ -0,0 +1,805 @@
+                        }]
+                    }]
+                  },{
+                    "conditions": ["false","true","web","true","false"],
+                    "results": [{
+                        "function": "excludeBidders",
+                        "args": [{
+                            "bidders": ["bidderA", "bidderB","bidderE"],
+                            "analyticsValue": "leaf6"
+                        }]
+                    }]
+                  },{
+                    "conditions": ["false","true","web","false","*"],
+                    "results": [{
+                        "function": "excludeBidders",
+                        "args": [{
+                            "bidders": ["bidderA", "bidderD","bidderF"],
+                            "analyticsValue": "leaf7"
+                        }]
+                    }]
+                  },{
+                    "conditions": ["false","true","*","*","true"],
+                    "results": [{
+                        "function": "excludeBidders",
+                        "args": [{
+                            "bidders": ["bidderA", "bidderC"],
+                            "analyticsValue": "leaf8"
+                        }]
+                    }]
+                  },{
+                    "conditions": ["false","true","*","*","false"],
+                    "results": [{
+                        "function": "excludeBidders",
+                        "args": [{
+                            "bidders": ["bidderA", "bidderF"],
+                            "analyticsValue": "leaf9"
+                        }]
+                    }]
+                  },{
+                    "conditions": ["false","false","web","*","*"],
+                    "results": [{
+                        "function": "excludeBidders",
+                        "args": [{
+                            "bidders": [],
+                            "analyticsValue": "leaf10"
+                        }]
+                    }]
+                  },{
+                    "conditions": ["false","false","*","true","*"],
+                    "results": [{
+                        "function": "excludeBidders",
+                        "args": [{
+                            "bidders": ["bidderB", "bidderE"],
+                            "analyticsValue": "leaf11"
+                        }]
+                    }]
+                  },{
+                    "conditions": ["false","false","*","false","*"],
+                    "results": [{
+                        "function": "excludeBidders",
+                        "args": [{
+                            "bidders": ["bidderE", "bidderF"],
+                            "analyticsValue": "leaf12"
+                        }]
+                    }]
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+And here's how these scenarios would play out in various runtime conditions:
```

Comment:
"And here's how these scenarios would play out in various runtime conditions given the following request:"

Then we need to add the request here with the two imps as that is missing.